### ISSUE TITLE
Using without custom domain name

### ISSUE THREAD

**Issue by nd7141** [huggingface/frp#8]
Is it possible to use this repo without creating a custom domain name, but instead using IP from EC2 instance directly?

**Comment by abidlabs** [comment]
I don't believe it's possible to just use an ip address as the `subdomain_host` key in the `frps.ini`: https://github.com/huggingface/frp/blob/tls/scripts/frps.ini
